### PR TITLE
Adding link to older OpenShift Vagrantfile as the new version not bac…

### PR DESCRIPTION
**Pull request by LalatenduMohanty**
…kward compatible

Signed-off-by: Lalatendu Mohanty lmohanty@redhat.com

### CODE REVIEW

File: components/centos/centos-openshift-setup/README.rst [hunk 1]
```diff
@@ -1,10 +1,12 @@
-Vagrantfile: CentOS Docker Base
-===============================
+Vagrantfile: OpenShift Setup In ADB Vagrant Box
+===============================================
```
Comment (navidshaikh) [inline components/centos/centos-openshift-setup/README.rst:2]: Would like to see the heading as

Vagrantfile: OpenShift setup in ADB Vagrant box

File: components/centos/centos-openshift-setup/README.rst [hunk 2]
```diff
@@ -1,10 +1,12 @@
...
 
 This Vagrantfile sets up the ADB for development with OpenShift.  This file also sets up private networking that will be used to expose various services to the host.  This Vagrantfile also provisions an instance of `OpenShift Origin <http://www.openshift.org//>`_.
 
 If you are interested in the process used in the Vagranfile to setup OpenShift, please read the comments at the top of the file.
 
+The Vagrantfile changes after ADB 1.6.0 is not backward compatible.  So if you are using ADB prior to 1.7.0, you can find the old `Vagrantfile here <https://github.com/projectatomic/adb-atomic-developer-bundle/blob/v1.6.0/components/centos/centos-openshift-setup/Vagrantfile>`_
```
Comment (navidshaikh) [inline components/centos/centos-openshift-setup/README.rst]: Vagrantfile(s) after ADB 1.6.0 are not backward compatible. So if you are using ADB prior to 1.7.0, you can find the respective Vagrantfiles using tags, for eg: Vagrantfile for ADB 1.6.0 https://github.com/projectatomic/adb-atomic-developer-bundle/blob/v1.6.0/components/centos/centos-openshift-setup/Vagrantfile

Comment (LalatenduMohanty) [inline components/centos/centos-openshift-setup/README.rst]: @navidshaikh it is just one Vagrantfile i.e. for OpenShift.

**Comment by navidshaikh** [comment]
LGTM

**Pull request merged**